controller/env/appenv.go

```diff
@@ -228,7 +227,7 @@ func (ae *AppEnv) ValidateServiceAccessToken(token string, apiSessionId *string)
 		return nil, err
 	}
 
```

Comment:
This is a bug fix. `tokenRevocation` is checked above and can only be nil here. I assume that this was supposed to be just `revocation`